ui/app/AppLayouts/Chat/ContactsColumn.qml

```diff
@@ -291,21 +291,70 @@ Item {
                         }
 
                         Rectangle {
-                            id: contactImage
+                          id: contactImage
+                          width: 40
+                          height: 40
+                          anchors.left: parent.left
+                          anchors.leftMargin: Theme.padding
+                          anchors.top: parent.top
+                          anchors.topMargin: 12
+                          anchors.bottom: parent.bottom
+                          anchors.bottomMargin: 12
+                          radius: 50
+
+                          Loader {
+                            sourceComponent: chatType == 1 ? imageIdenticon : letterIdenticon
```

Comment:
Is it possible to define constants somewhere like this:
```

readonly property int ChatTypeOneToOne: 1
readonly property int ChatTypePublic: 2
readonly property int ChatTypePrivateGroupChat: 3
```
and use them wherever `chatType` is being compared?